.github/workflows/release.yml

```diff
@@ -10,6 +10,8 @@ on:
         required: true
       skip_tests:
         description: 'Skip QA acceptance tests, define value to `true` to explicitly skip'
+      use_existing_tag:
```

Comment:
Can we add a small note in our release documentation on when this scenario could be useful or comes into picture?

Reply:
added